js/index.js

```diff
@@ -1 +1,69 @@
 // Your code goes here
+let logo = document.querySelector('.logo-heading');
+let body = document.querySelector('body');
+let busImg = document.querySelector('.intro img');
+let headings = document.querySelectorAll('h2');
+let button = document.querySelector('.btn')
+let textSelect = document.querySelector('textarea')
+let allImg = document.querySelectorAll('img');
+let navi = document.querySelector('nav')
+
+textSelect.style.width = '100%'
+
+logo.addEventListener('mouseover', function(event){
+    alert('you moused over my logo!')
+})
+
+body.addEventListener('keydown', function(){
+    body.style.backgroundColor = 'orange';
+})
+
+body.addEventListener('keyup', function(){
+    body.style.backgroundColor = 'white';
+})
+
+body.addEventListener('wheel', function(){
+    busImg.classList.toggle('red-border');
+})
+
+body.addEventListener('drag', function(){
+    headings.forEach(function (currentValue) {
+    currentValue.style.color = '#17A2B8';
+    })
+})
+
+textSelect.addEventListener('focus', function () {
+    textSelect.style.background = '#bfb';    
+});
+
+textSelect.addEventListener('select', function() {
+    alert('no selecting my text!')
+})
+
+button.addEventListener('dblclick', function(){
+    headings.forEach(function (currentValue) {
+    currentValue.textContent = 'you clicked my button!'
+    })
+})
+
+busImg.addEventListener('mouseenter', function() {
+    allImg.forEach(function (currentValue) {
+    currentValue.style.border = '2px solid yellow';
+    })
+})
+
+busImg.addEventListener('mouseleave', function() {
+    allImg.forEach(function (currentValue) {
+    currentValue.style.border = 'none';
+    })
+})
+
+navi.addEventListener('click', function(event) {
+    event.stopPropagation();
+    event.preventDefault();
```

Comment:
You only really need the one of these to ill the propagation over to the body. I would leave the `preventDefault` and get rid of the `stopPropagation`. That is the one that is more used and it also kills the propagation of the event as well.